src/lisp/regression-tests/framework.lisp

```diff
@@ -13,6 +13,22 @@
 (defparameter *test-marker-table* (make-hash-table))
 (defparameter *duplicate-tests* nil)
 
+(defun message (level control-string &rest args)
```

Comment:
may be worth splitting these test framework improvements into a separate, easy-to-approve PR, if thisone  takes a while because of cando or something.

Reply:
I think this is probably ready as is. I'm just being cautious in respect to Cando,

Reply:
Understood